build.sbt

```diff
@@ -101,7 +102,8 @@ lazy val fastparse = crossProject
         """.stripMargin
       IO.write(file, output)
       Seq(file)
-    }
+    },
+    libraryDependencies += "org.scodec" %%% "scodec-bits" % "1.1.0"
```

Comment:
Before publishing 0.4.0 I'd probably want to break out the `scodec-bits` dependency into a separate `fastparse-byte` subproject, so the core `fastparse` project doesn't pick up unnecessary dependencies